GCTC-NTGC/gc-digital-talent
# Allow v1 and v2 conditions of employment (alternative)

Resolves https://github.com/GCTC-NTGC/gc-digital-talent/issues/3387 (alt approach from https://github.com/GCTC-NTGC/gc-digital-talent/pull/3388)

Extracted from https://github.com/GCTC-NTGC/gc-digital-talent/pull/3249

the V1/V2 list refers to the subset of _inputs_ (the enum subsets), while the label variant key is more about "how do we show this" and is not necessarily time-specific (though it can be), e.g., we could create `shortV1` variant if there are still differences.

cc @vd1992 

To test:
- [x] Set the FEATURE_APPLICANTSEARCH flag to false and recreate your PHP container
- [x] Run refresh scripts
- [x] If necessary, run data seeders with `docker-compose exec -w /var/www/html/api php sh -c "php artisan db:seed"`
- [x] In one tab, open the search page (http://localhost:8000/en/talent/search) and find the Conditions of Employment options
- [x] In another tab, get to the Create Pool Candidate form:
    1. Go to http://localhost:8000/en/admin/pools
    2. Select "View Candidates" for a pool
    3. Click "Create Pool Candidate" in top right corner
- [x] In a third tab, get to the Update Pool Candidate form:
    1. Go to http://localhost:8000/en/admin/pools
    2. Select "View Candidates" for a pool
    3. Select "Edit" for a pool candidate
- [x] For both forms, the Operational Requirements should show the same options with the same text as on the Search page. The text should match in both English and French. They should be the options shown in https://github.com/GCTC-NTGC/gc-digital-talent/pull/3357.
